src/store/sagas/list.js

```diff
@@ -0,0 +1,92 @@
+import {
+  take,
+  takeEvery,
+  takeLatest,
+  call,
+  all,
+  put,
+} from "redux-saga/effects";
+import {
+  ALL_LIST,
+  ALL_LIST_SAGA,
+  DELETE_LIST,
+  DELETE_LIST_SAGA,
+  EDIT_LIST,
+  EDIT_LIST_SAGA,
+  ADD_LIST,
+  ADD_LIST_SAGA,
+} from "../types";
+import axios from "axios";
+const wait = (ms) => new Promise((res) => setTimeout(res, ms));
+
+//list
+export function* allList(actions) {
+  let url = "http://localhost:3001/list/all";
```

Comment:
Again url should be pick from environment variables.

Reply:
done